lively.ide/test-runner.js

```diff
@@ -308,19 +309,16 @@ export default class TestRunner extends HTMLMorph {
   // -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
 
   findBrowserForFile (file) {
-    const world = this.world();
-    const browsers = world.getWindows()
-      .map(win => win.targetMorph).filter(ea => ea.isBrowser);
-    const browserWithFile = browsers.find(({ selectedModule }) =>
-      selectedModule && selectedModule.url === file);
-    return browserWithFile || world.execCommand('open browser', null);
+    return browserForFile(file) || $world.execCommand('open browser');
   }
 
   async jumpToTest (test, file) {
     const li = LoadingIndicator.open('Scanning source code...');
     try {
       const browser = await this.findBrowserForFile(file);
-      browser.getWindow().activate();
+      const win = browser.getWindow();
+      if (win.minimized) win.minimized = false;
+      win.activate();
```

Comment:
Just a thought - since the minimization/maximization of windows is animated, do we need to await something here?